migrate/migrations.go

```diff
@@ -282,50 +283,112 @@ func (m *Migrations) MarkCompleted(ctx context.Context, db *bun.DB) error {
 	return nil
 }
 
-func (m *Migrations) Status(ctx context.Context, db *bun.DB) error {
-	allMigrations := m.sortedMigrations()
-	if len(allMigrations) > 0 {
-		fmt.Printf("In total you have %d migrations: %s\n",
-			len(allMigrations), migrationNames(allMigrations))
+type Status struct {
+	UpToDate bool
+	Locked   bool
```

Comment:
Let's remove these 2 vars?
- UpToDate seems to be unused and `len(s.NewMigrations) == 0` is good enough
- Locked seems unneeded too. I believe `Status` should return an error when table can't be locked.

Reply:
Sure. But, I would like to at least add `var ErrLocked = errors.New("migrate: migrations table is locked")` so that one could check if the table is locked. 

> It is an internal detail, not public interface.

Lock and Unlock functions are exported and used in the [bun-starter-kit](https://github.com/go-bun/bun-starter-kit/blob/0447f252b50e04e33ffeb8285092a507e06d1f80/cmd/bun/main.go#L124-L149), so it seems to me it kind of is. Unless I'm missing something.

Reply:
>Lock and Unlock functions are exported and used in the bun-starter-kit, so it seems to me it kind of is. Unless I'm missing something.

I mean that `Status` function may or may not lock the migrations so it is best to not make `Locked` part of the `Status` API. For example, in future we may decide to wait until lock is released.

>But, I would like to at least add 

Let's add it :+1: